tests/contrib/operators/test_gcp_vision_operator.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from google.api_core.exceptions import AlreadyExists
+from google.cloud.vision_v1.proto.product_search_service_pb2 import ProductSet, Product
+
+from airflow.contrib.operators.gcp_vision_operator import CloudVisionProductSetCreateOperator, \
```

Comment:
Here you can change the style to the one with brackets.

Reply:
Done